examples/api/lib/animation/animation_controller/animated_digit.0.dart

```diff
@@ -41,30 +50,31 @@ class _PlaceholderDigit extends StatelessWidget {
 // When the value changes the old value slides upwards and out of sight
 // at the same as the new value slides into view.
 class AnimatedDigit extends StatefulWidget {
-  const AnimatedDigit({ super.key, required this.value });
+  const AnimatedDigit({super.key, required this.value});
 
   final int value;
 
   @override
   State<AnimatedDigit> createState() => _AnimatedDigitState();
 }
 
-class _AnimatedDigitState extends State<AnimatedDigit> with SingleTickerProviderStateMixin {
+class _AnimatedDigitState extends State<AnimatedDigit>
+    with SingleTickerProviderStateMixin {
   static const Duration defaultDuration = Duration(milliseconds: 300);
 
   late final AnimationController controller;
   late int incomingValue;
   late int outgoingValue;
-  List<int> pendingValues = <int>[]; // widget.value updates that occurred while the animation is underway
+  List<int> pendingValues =
+      <
+        int
+      >[]; // widget.value updates that occurred while the animation is underway
```

Comment:
This one is ... odd. Is the expectation to place comments like this above the line to get sensible formatting?

Reply:
This is because of that long line comment. The formatter is desperately trying to get that line to fit in 80 columns but since it can't do anything with the comment, it splits in the only place it can, inside the type argument list.

If you move the comment to another line or shorten it, you'll get better output.

Reply:
> Can we now store the line length somewhere in a config file so the formatter automatically does the right thing?

Not currently, no. There is https://github.com/dart-lang/dart_style/issues/918. I'm now convinced it's worth doing, but haven't had the time to make it happen yet.